.aiox-core/data/tech-presets/angular-nestjs.md

```
@@ -0,0 +1,711 @@
+  const user = new UserBuilder().asAdmin().build();
+  authService['_user'].set(user);
+  fixture.detectChanges();
+  expect(fixture.nativeElement.querySelector('.admin-menu')).toBeTruthy();
+});
+```
+
+---
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)